src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -281,6 +279,17 @@ export type StatusMessage = {
     message: string
 };
 
+type oncokbCancerGene = {
+    oncokbAnnotated: boolean;
+    oncokbOcg: boolean;
```

Comment:
what do these two abbrevs mean?

Reply:
@zhx828 , I see, thanks for loop me in.

Reply:
@adamabeshouse this is oncogene. I will spell the full word to solve the confusion.